theme.go

```diff
@@ -29,6 +29,7 @@ var DefaultTheme = &Theme{
 		"list.item.selected":  {ColorWhite, ColorBlue},
 		"table.cell.selected": {ColorWhite, ColorBlue},
 		"button.focused":      {ColorWhite, ColorBlue},
+		"box.focused":         {ColorYellow, ColorDefault},
```

Comment:
I don't think the box should be highlighted by default. Instead it should be up to the user to set the colors by creating their own `Theme` and setting in to the `UI`.

```
	t := tui.NewTheme()
	t.SetStyle("box.focused", tui.Style{Bg: tui.ColorDefault, Fg: tui.ColorDefault})

	ui := tui.New(root)
	ui.SetTheme(t)
```

Removing this line should fix that.